src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -97,6 +100,11 @@ package object csv {
                       compressionCodec: Class[_ <: CompressionCodec] = null): Unit = {
       // TODO(hossein): For nested types, we may want to perform special work
       val delimiter = parameters.getOrElse("delimiter", ",")
+      // Before this change the csvFormatter wrote dates like this:
+      // "2014-11-15 06:31:10.0", so have that as the default.
+      val dateFormat = parameters.getOrElse("dateFormat", "yyyy-MM-dd HH:mm:ss.S")
```

Comment:
OK. I Added dateFormat property documentation to both the reading and writing sections.
Is there anything else, or can this be merged now?